wandb/data_types.py

```diff
@@ -39,15 +38,16 @@
 import codecs
 import tempfile
 from wandb import util
+from wandb.compat import tempfile
 
 
 # Get rid of cleanup warnings in Python 2.7.
-warnings.filterwarnings('ignore', 'Implicitly cleaning up', RuntimeWarning, 'backports.tempfile')
+warnings.filterwarnings('ignore', 'Implicitly cleaning up', RuntimeWarning, 'wandb.compat.tempfile')
 
```

Comment:
@raubitsj we should check the output of the regression to make sure these warnings are being filtered.

Reply:
./regression.py --cli_branch feature/jupyter --spec ::~broken main/
https://app.wandb.ai/jeffr/regression/groups/20190725-0.8.6-feature_jupyter-2496d15-pr0nem